build.gradle

```diff
@@ -34,6 +34,8 @@ dependencies {
 
     implementation 'uk.gov.digital.ho.hocs:hocs-ukvi-complaint-schema:1.1.1'
 
+    implementation 'uk.gov.digital.ho.hocs:hocs-migration-schema:0.4.0'
```

Comment:
Nitpick, could we move this to L36 so they're better grouped.

Reply:
Done.